Makefile

```diff
@@ -154,7 +153,10 @@ $(BUILD_DIR) $(DATA_DIR) $(DIST_DIR) $(DIST_DATA_DIR):
 	$(MKDIR) $@
 
 test: $(GIT_TEST_DATA_REPO_PATH)
-	$(TESTRUNNER) tests
+	@echo "[run]" > .coveragerc
+	@echo "omit = */invest/ui/*" >> .coveragerc
```

Comment:
I originally did this in one line with an `\n` using `echo -e`, but the windows tests were failing because the `-e` flag wasn't recognized.

Reply:
Interesting!  Although it looks like the Windows tests are still failing with an exception ([example line](https://github.com/natcap/invest/runs/942585849?check_suite_focus=true#step:12:260)), perhaps related to the encoding of the file?

Reply:
Oh! I had added `-` to the line running the tests so it would continue to the next line and remove `.coveragerc` whether the tests pass or fail. Just realized that is a really bad idea since the tests will always appear to pass!